packages/xod-client/src/project/nodeLayout.js

```diff
@@ -21,6 +21,7 @@ export const LINK_HOTSPOT_SIZE = {
 export const NODE_CORNER_RADIUS = 5;
 
 export const PIN_RADIUS = 6;
+export const PIN_INNER_RADIUS = PIN_RADIUS - 3;
```

Comment:
I think it’s better to use 2px border rather than 3 because non-linked pins have a 2px stroke.